rq/compat/__init__.py

```diff
@@ -103,4 +103,13 @@ def tzname(self, dt):
         def dst(self, dt):
             return timedelta(0)
 
-    utc = UTC()
\ No newline at end of file
+    utc = UTC()
+
+
+def hmset(pipe_or_connection, name, mapping):
+    # redis-py versions 3.5.0 and above accept a mapping parameter for hset
+    try:
+        return pipe_or_connection.hset(name, mapping=mapping)
```

Comment:
This [signature](https://github.com/andymccurdy/redis-py/pull/1271/files#diff-085cbe85ef9bc2ad832d79163eb39172R2999) for `hset` changed in `redis-py` 3.5.0, so this will immediately raise a `TypeError` if it's wrong.

Reply:
An alternative would be to check `redis.__version__`.